bin/recipes/001-import.js

```diff
@@ -30,14 +32,13 @@ let options = {
 	raven: raven
 }
 
-
 async.parallel({
 		users       : async.apply(helper.create, options, Users),
 		recipes     : async.apply(helper.create, options, Recipes),
 		menus       : async.apply(helper.create, options, Menus),
 
 		// for recipes 2
-		ingr-ex     : async.apply(helper.create, options, IngrEx),
```

Comment:
It didn't want to accept dashes when I tried to run the import,, so had to remove the dash

Reply:
not sure what do you mean by dashes?

Reply:
This is the error I get:

![image](https://user-images.githubusercontent.com/16187649/40652273-21548534-6338-11e8-8080-804df253b7f7.png)

Reply:
rename it to `ingrEx` or `ingr_ex`. maybe node have some restrictions in variable naming

Reply:
cool, it was already renamed to ingrEx in this pull request. I just changed it back in my copy for demonstration purposes.